README.md

```diff
@@ -1,24 +1,80 @@
-**STILL IN DEVELOPMENT, NOT FIT FOR GENERAL CONSUMPTION, DO NOT USE IF NOT A DEVELOPER!**
-
 # Slic3r plugin for OctoPrint
 
-**WARNING** Not functional yet since slic3r's return codes are behaving unexpectedly. WIP for others to take a look at
-and maybe build upon.
+This plugin has been successfully tested for Slicer's stable versions 1.1.7 and 1.2.9.
 
 ## Setup
 
-Install the plugin like you would install any regular Python package from source:
+In order to install the plugin, go to Settings -> Plugin Manager and click on 'Get more...':
+
+![Screenshot](http://imgur.com/9NaAl37.png)
+
+You'll see an option to add it from an URL. Add https://github.com/javierma/OctoPrint-Slic3r/archive/master.zip an click on 'Install' button.
```

Comment:
Wrong URL for this repository, please adjust to github.com/OctoPrint/OctoPrint-Slic3r (and if that stuff can be considered stable, it can be added to the repository too)